### PR TITLE
added support e-mail

### PR DESCRIPTION
**Pull request by wslack**
Is this the right place to add support e-mail for logged in users only?

cc @saracope

### CODE REVIEW

File: assets/app/components/nav.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ const getSecondaryNav = (username) => {
   const links = [
     <Link to="/sites">{username}</Link>,
     <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
-    <a href="https://github.com/18F/federalist/issues/new" target="_blank">Contact us</a>,
+    <a href="https://github.com/18F/federalist/issues/new" target="_blank">File Issue</a>,
+    <a href="mailto:federalist-support@gsa.gov" target="_blank">Get Help</a>,
```
Comment (jmhooper) [inline assets/app/components/nav.js]: This looks like it should work. I don't know that you need to `target="_blank"` though?

Comment (wslack) [inline assets/app/components/nav.js]: yeah I just copy-pasted. Didn't we used to haev different links for logged in and not logged in users?

Comment (jmhooper) [inline assets/app/components/nav.js]: We do. Unauthed users will see [these](https://github.com/18F/federalist/blob/staging/assets/index.html#L57-L59)

Comment (wslack) [inline assets/app/components/nav.js]: great!

Comment (wslack) [inline assets/app/components/nav.js]: Made the change you asked for

**Comment by wslack** [comment]
Confirmed this is working on staging.

**Pull request merged**